### PR TITLE
sso mission updates

### PR DESCRIPTION
**Pull request by lavernw1**
WIP: DO  NOT MERGE: Fixes[ RHOARDOC-124 ](https://issues.jboss.org/browse/RHOARDOC-124)(github issue 978)  Minimalism feedback - simplify writing  
 @rhoads-zach @Ladicek @tradej @inoxx03  Please take a look a sso mission changes. Posting subsequent changes for remaining missions next. simplified paragraphs,  Moved informational text  into into procedures.

### CODE REVIEW

File: docs/topics/note-booster-unvailable-oso.adoc [hunk 1]
```diff
@@ -1 +1,3 @@
-IMPORTANT: This booster is not currently available on {OpenShiftOnline} Starter. You can still run it using a {OpenShiftLocal}. You can also use a manual workflow to deploy this booster to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
\ No newline at end of file
+//IMPORTANT: This booster is not currently available on {OpenShiftOnline} Starter. You can still run it using a {OpenShiftLocal}. You can also use a manual workflow to deploy this booster to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
+*Limitation:* Run this booster on a {OpenShiftLocal}. You can also use a manual workflow to deploy this booster to {OpenShiftOnline} Pro and {OpenShiftContainerPlatform}.
```
Comment (Ladicek) [inline docs/topics/note-booster-unvailable-oso.adoc]: Nit: multiple sentences on single line :-)

Comment (Ladicek) [reply docs/topics/note-booster-unvailable-oso.adoc]: Ah nevermind, I see it's on multiple places, so the "one line = one sentence" convention probably doesn't apply in this repo. Sorry about the noise.

**Comment by Ladicek** [comment]
LGTM, I guess :-)

**Comment by rhoads-zach** [comment]
merged: 3d00ead829f2ff58110e10a0df799ae0a5bf00bb

**Pull request closed without merging**